backend/src/main/java/com/cs471/studentLoanSystem/sql/descriptions/Loan.java

```diff
@@ -0,0 +1,83 @@
+package com.cs471.studentLoanSystem.sql.descriptions;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+enum LoanStatus {
+    APPLIED,
+    APPROVED,
+    PAID
+}
+
+@SuppressWarnings("unused")
+@Entity
+public class Loan {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Integer id;
+
+    private Integer bankId;
+    private Float loanAmount;
+    private Float loanInterest;
+    private String loanTerms;
+    private LoanStatus loanStatus;
+    private Integer studentId;
```

Comment:
Which of these types are nullable in the database? If a value isn't nullable it should remain a primitive I think.

Reply:
I messaged bolar, waiting for a response.

Reply:
Fixed the non-nullable values